tests/test_extensions/test_sklearn_extension/test_sklearn_extension.py

```diff
@@ -161,11 +168,18 @@ def test_serialize_model_clustering(self):
         with mock.patch.object(self.extension, "_check_dependencies") as check_dependencies_mock:
             model = sklearn.cluster.KMeans()
 
-            fixture_name = "sklearn.cluster.k_means_.KMeans"
+            cluster_name = "k_means_" if LooseVersion(sklearn.__version__) < "0.22" else "_kmeans"
+            fixture_name = "sklearn.cluster.{}.KMeans".format(cluster_name)
             fixture_short_name = "sklearn.KMeans"
             # str obtained from self.extension._get_sklearn_description(model)
-            fixture_description = "K-Means clustering"
+            fixture_description = "K-Means clustering{}".format(
+                "" if LooseVersion(sklearn.__version__) < "0.22" else "."
+            )
             version_fixture = "sklearn==%s\nnumpy>=1.6.1\nscipy>=0.9" % sklearn.__version__
+
+            n_jobs_val = "null" if LooseVersion(sklearn.__version__) < "0.23" else '"deprecated"'
```

Comment:
This could be further extended so that we can get rid of the if/else statement below which is also about n_jobs, right?

Reply:
Are you referring to the if-else [here](https://github.com/openml/openml-python/blob/dcf2d22e4b92b6afc5bfb15b69c4f8d37f15d57a/tests/test_extensions/test_sklearn_extension/test_sklearn_extension.py#L184)?

If so, then I added L180 to avoid an `elif` or a nested structure since the if-else block differentiates three divisions of sklearn versions --- "< 0.20", "<0.23", and ">=0.23".

Did I address the issue?

Reply:
Yes, I do - this looks much simpler now.